client/src/app/api/rest.tsx

```diff
@@ -478,7 +478,17 @@ export const deleteBulkApplications = (ids: number[]) =>
   axios.delete(APPLICATIONS, { data: ids });
 
 export const getApplications = () =>
-  axios.get<Application[]>(APPLICATIONS).then((response) => response.data);
+  axios
+    .get<Application[]>(APPLICATIONS)
+    .then((response) => response.data)
+    .catch((error: Error | AxiosError) => {
+      if (axios.isAxiosError(error)) {
+        return error;
+      } else {
+        console.log(`${error}`);
+        return;
```

Comment:
We should still be throwing the error so that it can be picked up by our errorBoundary component. Also - We should be implementing this global error handling for non axios errors at the global axios interceptor level rather than as a one off implementation for applications.